Assets/MRTK/SDK/Experimental/Features/UX/BoundsControl/Visuals/Configuration/TranslationHandlesConfiguration.cs

```diff
@@ -0,0 +1,123 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using Microsoft.MixedReality.Toolkit.Experimental.UI.BoundsControlTypes;
+using UnityEngine;
+using UnityEngine.Events;
+
+namespace Microsoft.MixedReality.Toolkit.Experimental.UI.BoundsControl
+{
+    /// <summary>
+    /// Configuration for <see cref="TranslationHandles"/> used in <see cref="BoundsControl"/>
+    /// This class provides all data members needed to create translation handles for <see cref="BoundsControl"/>
+    /// </summary>
+    [CreateAssetMenu(fileName = "TranslationHandlesConfiguration", menuName = "Mixed Reality Toolkit/Experimental/Bounds Control/Translation Handles Configuration")]
+    public class TranslationHandlesConfiguration : HandlesBaseConfiguration
```

Comment:
Reminder; this is mostly identical to the existing `RotationHandlesConfiguration` and `ScaleHandlesConfiguration`. Suggestion for this file should also apply to the other files, too.